patricia/patricia.go

```diff
@@ -19,6 +19,29 @@ type (
 	VisitorFunc func(prefix Prefix, item Item) error
 )
 
+const (
+	DefaultMaxPrefixPerNode         = 10
+	DefaultMaxChildrenPerSparseNode = 8
+)
+
+type Option struct {
+	// Max prefix length that is kept in a single trie node.
+	MaxPrefixPerNode int
+	// Max children to keep in a node in the sparse mode.
+	MaxChildrenPerSparseNode int
+}
+
+func NewOption(maxPrefixPerNode int, maxChildrenPerSparseNode int) *Option {
```

Comment:
I see. The initial motivation why I want to change MaxChildrenPerSparseNode is when sparseChildList is used, Visit function doesn't apply VisitorFunc in order of Prefix (it applies VisitorFunc in order of addition). Is this right behavior?  I want to apply VisitorFunc in order of Prefix.